scss/civicrm/contact/_tabs.scss

```diff
@@ -164,6 +165,7 @@
     .ui-tabs-panel {
       box-sizing: border-box;
       float: right;
+      padding: 20px 20px 0 0 !important;
```

Comment:
why did you add this padding?

Reply:
Previously we were doing it tab by tab to add padding on left and top: and hence, majority of the tabs were missing this spacing, I made it global to make it look proper.

without it:
![image](https://user-images.githubusercontent.com/26058635/37821455-a3edd3ec-2ea9-11e8-9303-b0c46f4665c4.png)